dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CDIBeanInterceptorServletTest.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
```

Comment:
You can remove the contributors section of the copyright header as well.

Reply:
I won't comment on each file but there are others that could be cleaned up. Thanks!